pkg/encoding/v1/encoding.go

```diff
@@ -233,16 +302,37 @@ func (s *Service) UnmarshalYAML(unmarshal func(interface{}) error) error {
 
 type VolumeSize string
 
-// FIXME: add VolumeSize parsing/validation
+func (vs *VolumeSize) UnmarshalYAML(unmarshal func(interface{}) error) error {
```

Comment:
I did it as it's done for `ResourceName` [here](https://github.com/redhat-developer/opencompose/blob/59729bacb6849ecc369e15cc965b4b8214dbdc51/pkg/encoding/v1/encoding.go#L21).